dask/bag/tests/test_random.py

```diff
@@ -48,7 +48,7 @@ def test_choices_k_equal_bag_size_with_unbalanced_partitions():
     seq = range(10)
     sut = db.from_sequence(seq, partition_size=9)
     li = list(random.choices(sut, k=10).compute())
-    assert sut.map_partitions(len).compute() == (9, 1)
+    assert sut.map_partitions(len).compute() == [9, 1]
```

Comment:
the bag output type has been a bit inconsistent so far depending on whether the output is a single partition or not (which in itself is not trivial for bags). Now, the output is always a list